src/modules/get_institution/app/get_institution_controler.ts

```diff
@@ -0,0 +1,57 @@
+import { GetInstitutionUsecase } from "./get_institution_usecase";
+
+import { EntityError } from '../../../core/helpers/errors/EntityError';
+import { Conflict, OK, Unauthorized, BadRequest, HttpRequest, HttpResponse} from '../../../core/helpers/http/http_codes';
+import { ConflictError, InvalidParameter, InvalidRequest, MissingParameter, UserNotAuthenticated } from '../../../core/helpers/errors/ModuleError';
+import { ParameterError, InternalServerError } from '../../../core/helpers/http/http_codes';
+
+
+export class GetInstitutionController {
+    public usecase: GetInstitutionUsecase;
+
+    constructor(usecase: GetInstitutionUsecase) {
+        this.usecase = usecase;
+    }
+
+    public async execute(request: HttpRequest): Promise<HttpResponse> {
+        try {
+            if(!request)
+                {
+                    throw new InvalidRequest();
+
+                }
+            if(!request.headers)
+                {
+                    throw new InvalidRequest("Headers");
+
+                }
+            if(!request.body){
+                throw new InvalidRequest("Body");
+            }
+
+            let response = await this.usecase.execute(request.body.body ,request.headers);
```

Comment:
here is the queryStringParameters, because is a get method so pass to usecase the request.body.queryStringParameters